aea/manager/manager.py

```diff
@@ -175,10 +181,102 @@ def stop(self,) -> None:
             self._thread.join()
 
 
+class AgentRunProcessTask(AgentRunAsyncTask):
```

Comment:
You're not using anything of `AgentRunAsyncTask` afaikt. So why not have a separate class for the process case and keep backwards compatibility above i.e. revert changes above.

Reply:
updated. added common base class for agent run tasks